### DISCUSSION TITLE
Configuration of several Panels

### DISCUSSION THREAD

**Discussion by luckiesdavid** [Blackymas/NSPanel_HA_Blueprint#1072] [Ideas]
Hello,

i have now 3 nspanels, and maybe more in the future an have the Problem that i have already different configurations.
Most of my Sensor could be the same, but some are specific for every panel.

For this i have made me a local python Script where i scan the "main" Panel and overwrite the specific Sensor of the other Panels.
but this is local and not nice to use..

I think the best idea were when it will be possible to have a "master" blueprint with the specifics at each panel, but i don't sure if this will be the best solution, any other ideas? ;) 

Greetings David

**Comment by edwardtfn** [comment]
Yaml offers a range of options like anchors, alias, overriding, `!import`, etc... you probably will find some way to make this a bit easier and still supported by Home Assistant.
I understand there's no UI for this, but I believe it isn't that hard to get there playing with your `automations.yaml`.

https://www.educative.io/blog/advanced-yaml-syntax-cheatsheet

**Comment by luckiesdavid** [comment]
thanks for your reply, i give it a try if this will be a solution :)

**Comment by luckiesdavid** [comment]
If i use this anker for the "master" Blueprint it works, also its possible to set the specific Sensor for each Panel..
but the Problem is now that, when i want to use the GUI for the Master Panel it deletes the Anker "&master"
why is this happening? :)
This is how its looks and works for me, but only in .yaml File.
<img width="341" alt="image" src="https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/114309176/90d11cb6-826b-4d2d-9691-ee0678785d4c">

<img width="343" alt="image" src="https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/114309176/e07b2099-b046-4138-bc5a-84bc2825cbdc">

**Reply by edwardtfn** [reply]
I don't think UI mode will support anchors. 😩
